### ISSUE TITLE
Does mosh has a command to save(dump) a buffer history to a file?

### ISSUE THREAD

**Issue by cmal** [mobile-shell/mosh#966]
I've found in the test tmux's `save-buffer` has been tested.

https://github.com/mobile-shell/mosh/blob/7c68667bc0cdb3f463d3dc25557f2d7426305335/src/tests/e2e-test-server#L82

Does mosh provide a command to save the buffer content itself?

Thanks!

**Comment by cgull** [comment]
No, Mosh doesn't have any functionality like that (which is why we use tmux).  It's certainly not impossible to add, but tmux covers our testing needs, and nobody's expressed a need for this elsewhere.  Is this something you're looking for?

**Comment by cmal** [comment]
Thank you for replying. I am still trying to use mosh with tmux. I've found mosh works very much differently from ssh. For example, when I change the iTerm2 window size, it will not redraw the mosh session's output, but if I used ssh, it will redraw.
![image](https://user-images.githubusercontent.com/1773318/38655142-8e4a34c6-3e45-11e8-9a69-47add9d42ea6.png)
(Before the `cat` command, I switch the window to full-screen. The output before that are not redrawed. This will not be true if I am using ssh).
